cmd/clone/clone.go

```diff
@@ -45,7 +45,7 @@ func NewCloneCmd() *cobra.Command {
 		Run:   run,
 	}
 
-	cmd.Flags().BoolVar(&nofork, "no-fork", false, "Will not fork, just clone and create a branch.")
+	cmd.Flags().BoolVar(&forceFork, "fork", false, "Force forking, instead of turbolift attempting to branch push")
```

Comment:
```suggestion
	cmd.Flags().BoolVar(&forceFork, "fork", false, "Force forking, instead of turbolift choosing whether to fork/branch based on permissions")
```